### ISSUE TITLE
Binary candidates for `put`, `get`

### ISSUE THREAD

**Issue by jonathanstowe** [bduggan/aws-s3-raku#5]
Right now it's not possible to deal with "binary" files such as images and so forth, and it would be nice to do so.

Given that `HTTP::UserAgent` deals quite well with binary payloads I don't think this would be that hard to do and if I get a minute I'll take a look.

**Comment by bduggan** [comment]
True!  Thanks for bringing this up -- would be great to support this.

**Comment by kawaii** [comment]
+1 for this, would be extremely useful for one of my upcoming projects. :)
